lib/Crypt/Argon2.xs

```diff
@@ -65,7 +65,7 @@ argon2i_raw(t_cost, m_cost, parallelism, password, salt)
 		SvREFCNT_dec(RETVAL);
 		Perl_croak(aTHX_ "Couldn't compute argon2i hash: %s", argon2_error_message(rc));
 	}
-	SvCUR(output) = 32;
```

Comment:
This is a refactor screw-up by me, instead of casting the line should be removed (this function isn't tested yet currently)